skyportal/models/user_token.py

```diff
@@ -25,10 +25,22 @@
 
 
 def basic_user_display_info(user):
-    return {
+    user_info = {
         field: getattr(user, field)
         for field in ('username', 'first_name', 'last_name', 'gravatar_url')
     }
+    if getattr(user, 'preferences') and ('avatar' in user.preferences):
+        user_info['avatar'] = user.preferences['avatar']
+    else:
+        user_info['avatar'] = None
+    return user_info
+
+
+def user_avatar(user):
+    if getattr(user, 'preferences') and ('avatar' in user.preferences):
+        return user.preferences['avatar']
+    else:
+        return None
```

Comment:
same as above. Actually I would just replace the code above with a call to `user_avatar`.

Reply:
ah yes.